script.module.pylast/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pylast" version="1.8.0" name="pyLastFM" provider-name="Lunatixz">
```

Comment:
note that when porting other devs python modules to python script modules you are required to give credits to the proper author. As long as the code is the same and this is not a fork of the module the original author should remain as the author of the code.